leits/MeetingBar
# Feat/183 outlook safe urls

### Status
**READY**

### Description
We will now recognize outlook safe links -> we will extract and decode the original link, so that the link can be handled with the original logic to detect a meeting link 

### Steps to Test

- Create a meeting with a safe url in it (see the original ticket)
- Check that the real meeting service is detected and shown up in the systembar and menu.
